README.md

```diff
@@ -181,6 +181,23 @@ Class names like `Options` and `Level` might be too common, if you get a conflic
     })
     ...
 
+#### Angular Universal
+
+To use it with Angular Universal add angular2-logger to App.module normaly and in node.module provide these custom options.   
+
+```javascript
+import { Options } from "angular2-logger/core";
+
+@NgModule({
+  bootstrap: [ AppComponent ],
+  imports: [ AppModule, ... ],
+  providers: [
+    { provide: Options, useValue: { global: false, store: false } }, // stop angular2-logger from defining global variables.
```

Comment:
@langley-agm , not sure what you meant by that ? With Angular Universal the HTML is generated both in the server and in the client. 
https://medium.com/google-developer-experts/angular-universal-for-the-rest-of-us-922ca8bac84#.kr0acugau